flox-bash/lib/commands/activate.sh

```diff
@@ -286,6 +286,14 @@ function floxActivate() {
 		;;
 	esac
 
+	# Earlier parts of the activation script set individual `FLOX_ENV' vars
+	# interspersed between `shell.hook' bodies.
+	# Because the "target" environment may not be the last to activate, we
+	# explicitly set `FLOX_ENV' here so that once activated the variable will
+	# point to the environment indicated on the command line.
```

Comment:
How does this work if multiple environments are specified on the command line?

Reply:
it'll use the first I suppose.

Reply:
Can you clarify that in the comment?

Reply:
I mean that running `flox activate -e foo -e bar` will cause `FLOX_ENV` to be set to the prefix of `foo`.

Not ideal; but I think it's fine. The actually important part is that these are set properly when activation hooks are run.

Reply:
```suggestion
	# explicitly set `FLOX_ENV' here so that once activated the variable will
	# point to the first environment indicated on the command line.
```